api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
+   "version": "v1.6.0"
```

Comment:
I couldn't commit any changes without this change. This is how it goes:
------------------------------------------------
I run ./hack/update-staging-client-go.sh
which gives me:
“Commit your changes in another terminal and then continue here by pressing enter.
“ 
I run “git commit -a” and that gives me:
ERROR!
Openapi spec needs to be updated.
To regenerate the spec, run:
  hack/update-openapi-spec.sh

ERROR!
Federation OpenAPI spec needs to be updated.
To regenerate the spec, run:
  hack/update-federation-openapi-spec.sh

Aborting commit
--------------------------------------------
I am trying to cheat if can 'commit' without the swagger version change, let's see if it works. Thanks!!